### ISSUE TITLE
bug(helpers): convertLength with yards is not correct

### ISSUE THREAD

**Issue by lisabaut** [Turfjs/turf#1857]
## Problem: helper `convertLength` with unit yards

The following code
```TS
convertLength(1, 'meters', 'yards'); // =>  0.91441111923921
convertLength(1, 'yards', 'miles'); // => 0.0006795315358307484
```
returns incorrect values.

1 meter equals `1.0936132983` yards. 
Instead the helper returns `0.91441111923921` yards.

1 yards equals `0.0005681818` miles. 
Instead the helper returns `0.0006795315358307484` miles.

Same issue with trying to convert any other Unit with/from yards.

As I did not find any open/closed issue on this topic, I am wondering if I have missed something with the correct usage?

Thanks

**Comment by rowanwins** [comment]
Thanks for the report @lisabaut - perhaps your the first person ever to calculate a distance in yards 😂

**Comment by ajf2** [comment]
There's an open PR for this here: https://github.com/Turfjs/turf/pull/1837

**Comment by lisabaut** [comment]
Just saw another PR which is merged to master:
https://github.com/Turfjs/turf/pull/2046

**Comment by rowanwins** [comment]
Resolved by release of v6.4.0
